static/app/views/issueDetails/groupTags.tsx

```diff
@@ -102,6 +110,20 @@ function GroupTags({group, baseUrl, environments, event}: GroupTagsProps) {
     );
   }
 
+  const getTagKeyTarget = (tag: SimpleTag) => {
+    return {
+      pathname: generateTagsRoute({orgSlug: organization.slug}),
+      query: {
+        ...extractSelectionParameters(location.query),
+        start: (beforeDateTime as Date).toISOString(),
+        end: (afterDateTime as Date).toISOString(),
+        statsPeriod: undefined,
+        tagKey: tag.key,
+        transaction,
```

Comment:
These should only be applied when it's a statistical detector issue. You can check that by doing `group.issueType === IssueType.PERFORMANCE_DURATION_REGRESSION`

Reply:
So should the should the pathname be the older one too if it's not a stats detector issue?

Reply:
Yeah, that's right

Reply:
Fixed logic.